lambda/sendReminder/index.js

```diff
@@ -0,0 +1,203 @@
+          let jobObj = await postBulkReminderSummary(resData, jobError, chunk);
+          if (jobError) {
+            // Post alert to RocketChat channel if job fails.
+            request = {
+              postTitle: '@all **Day Use Pass - Bulk Email Service**',
+              postText: `A bulk email reminder job has failed: ${jobError} Details can be found under the job key listed below.`,
+              author_name: 'Day Use Pass',
+              author_icon: 'https://bcparks.ca/_shared/images/logos/logo-bcparks-v-200.png',
+              color: '#2D834F',
+              fields: [
+                {
+                  title: 'Job key:',
+                  value: `pk: ${jobObj?.pk}\nsk: ${jobObj?.sk}`,
+                  short: true
+                },
+                {
+                  title: 'Number of users affected',
+                  value: `${chunk.length - 1}`,
+                  short: true
+                }
+              ]
+            }
+            await rcPost(process.env.RC_ALERT_WEBHOOK_URL, process.env.RC_ALERT_WEBHOOK_TOKEN, request);
+          }
+        } catch (err) {
+          logger.error('Failed to document the bulk email job:', err);
+        }
+      }
+    } else {
+      logger.debug('No passes found for reminder service.')
+    }
+    const totalJobs = bulkJobSuccesses + bulkJobFailures;
+    logger.debug(`${totalJobs} job(s) run (${bulkJobSuccesses} succeeded, ${bulkJobFailures} failed).`)
+  } catch (err) {
+    // Something unknown went wrong.
+    return sendResponse(400, { msg: 'Something went wrong.', title: 'Operation Failed' });
+  }
+}
+
+// Construct pass cancellation links to include in reminder emails.
+function buildCancellationLink(pass) {
+  const cancellationLink = process.env.PUBLIC_FRONTEND +
+    process.env.PASS_CANCELLATION_ROUTE +
+    '?passId=' +
+    pass.sk +
+    '&email=' +
+    pass.email +
+    '&park=' +
+    pass.pk.split('::')[1] +
+    '&date=' +
+    pass.shortPassDate +
+    '&type=' +
+    pass.type;
+  return cancellationLink;
+}
+
+async function postBulkReminderSummary(data, jobError, passArray) {
+  let passes = [];
+  let postItem;
+  if (passArray && jobError) {
+    // We want a list of passes.
+    for (let i = 1; i < passArray.length; i++) {
+      passes.push(passArray[i][5]);
+    }
+  }
+  try {
+    postItem = {
+      pk: 'sendReminderSummary',
+      sk: DateTime.utc().toISO(),
+      status: jobError ? 'fail' : 'success',
+      response: data,
+      passes: passes,
+    };
+    let postObj = {
+      TableName: EXTRAS_TABLE_NAME,
+      Item: AWS.DynamoDB.Converter.marshall(postItem),
+      ConditionExpression: 'attribute_not_exists(pk) AND attribute_not_exists(sk)',
+    };
+    await dynamodb.putItem(postObj).promise();
+    logger.debug('Posted bulkReminderSummary to database:', postItem);
+    return postItem;
+  } catch (err) {
+    logger.error('Failed to save bulkReminderSummary to database:', err);
+    return postItem;
```

Comment:
Same here - lets emit to RC since you're not throwing.

Reply:
Addressed in `a508c34`.